caddytls/maintain.go

```diff
@@ -334,8 +334,15 @@ func DeleteOldStapleFiles() {
 // meaning that it is not expedient to get an
 // updated response from the OCSP server.
 func freshOCSP(resp *ocsp.Response) bool {
+	nextUpdate := resp.NextUpdate
+        // If there is an OCSP responder certificate, and it expires before the
```

Comment:
Ah, now the CI tests are complaining about the formatting. Needs `gofmt` - i.e. Comments need to be tab-indented and aligned with the rest of this block. Sorry I didn't catch this before. 😇